### PR TITLE
[JENKINS-65161] remove commons-digester2 from core and upgrade plugins to commons-digester3

### PR DESCRIPTION
**Pull request by olamy**

### CODE REVIEW

**Comment by olamy** [comment]
https://github.com/jenkinsci/jenkins/pull/5320

**Review by knapsu — APPROVED** [review]
Simple changes. Looks good to me.

**Review by daniel-beck — CHANGES_REQUESTED** [review]
Simple and wrong: Likely creates a security vulnerability as described in https://github.com/jenkinsci/jenkins/pull/5320#issuecomment-809979530

**Comment by olamy** [comment]
@knapsu  @daniel-beck fixed

File: pom.xml [hunk 1]
```diff
@@ -121,6 +121,17 @@
     </pluginRepository>
   </pluginRepositories>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>commons-beanutils</groupId>
+        <artifactId>commons-beanutils</artifactId>
+        <!-- 1.9.4 when this merged https://github.com/stapler/stapler/pull/211 -->
```
Comment (rsandell) [inline pom.xml:129]: Seems to be released in stapler-parent-1.262.1

**Review by rsandell — APPROVED** [review]

**Comment by olamy** [comment]
@mig42 @knapsu @miryamGSM ping (as you are the main maintainers)

**Comment by mig42** [comment]
Thanks for finishing the PR! We'll get it merged ASAP, when we can allocate some time to also update the code in our internal repositories.

**Review by alecharp — APPROVED** [review]

**Review by MRamonLeon — APPROVED** [review]

**Review by bitwiseman — DISMISSED** [review]

**Review by alecharp — APPROVED** [review]

**Comment by alecharp** [comment]
@mig42 is there any blockers for a merge and release of the plugin? if so, please let us know, so we can try to help you.

**Review by MRamonLeon — APPROVED** [review]

**Review by jtnord — APPROVED** [review]

**Comment by olamy** [comment]
@mig42 ping :)

**Review by mig42 — APPROVED** [review]
Changes look good to me. I'll give it a spin in my environment and merge it if everything works as expected.

**Pull request merged**